Planner/src/Planner.java

```diff
@@ -227,7 +367,23 @@ public void actionPerformed(ActionEvent arg0) {
 			        		TaskDAO.insertTask(t, users); 
 			        	}
 			         }
-		}}});
+			         
+			      // save tasks repeatable monthly
+			         if (repeatableMonthly.isSelected()) {
+						int month = taskCreated.getStartTime().getMonth();
+						for (int i = 1; i <= 12; i++) {
+							Task t = new Task(taskName.getText(), taskDescription.getText()); 
+							Date startDate = taskCreated.getStartTime();
+							startDate.setMonth(month + i);
```

Comment:
This code here is not quite good. Imagine that you create a task with startDate = 12/05/2018. Here you create a for from 1 to 12 => when i = 12 you will add 12 to your month => this will become month=17 and this does not exist and there will be exceptions. You should do the for not untill 12 but untill (i<= 12 - month).